src/Editor/src/Scripting/PythonInterface.cpp

```
@@ -125,11 +132,36 @@ static PyObject *osre_project_close(PyObject*, PyObject*, PyObject*) {
+    if (gActiceProject == nullptr) {
+        return nullptr;
+    }
+    ImportAction action(nullptr, nullptr);
+    Py_RETURN_NONE;
+}
+
+PyDoc_STRVAR(osre_project_export_doc, "ToDo.");
+static PyObject *osre_project_export(PyObject *, PyObject *, PyObject *) {
+    if (gActiceProject == nullptr) {
+        return nullptr;
+    }
```

Cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)